README.md

```diff
@@ -352,7 +352,7 @@ Populated after `ready` has been emitted. Will be `null` before the event.
 
 #### `feed.discoveryKey`
 
-Buffer containing a key derived from the feed.key.
+Buffer containing a key derived from the feeds public key.
```

Comment:
I think feeds should have an apostrophe on the end to be grammatically correct, as it is possessive. Ie. `from the feeds' public key`